pkg/ccl/streamingccl/streamclient/client.go

```diff
@@ -64,7 +69,7 @@ type Client interface {
 
 	// Plan returns a Topology for this stream.
 	// TODO(dt): separate target argument from address argument.
-	Plan(ctx context.Context, streamID streaming.StreamID) (Topology, error)
+	Plan(ctx context.Context, streamID streaming.StreamID) (*jobspb.StreamTopology, error)
```

Comment:
Not sure if this should be a pointer or not, I'm not sure how the tradeoff works in Go, would using a pointer save some unnecessary copies if we never mutate this, or would the compiler already optimize away extra copies for us?